cmd/argoexec/commands/wait.go

```diff
@@ -45,6 +47,18 @@ func waitContainer(ctx context.Context) error {
 		wfExecutor.AddError(err)
 	}
 
+	if wfExecutor.Template.Resource != nil {
+		// Save log artifacts for resource template
+		artifacts := wfv1.Artifacts{}
+		logArtifacts := wfExecutor.SaveLogs(bgCtx)
+		artifacts = append(artifacts, logArtifacts...)
+		err = wfExecutor.ReportOutputsLogs(bgCtx, artifacts)
+		if err != nil {
+			wfExecutor.AddError(err)
+		}
+		return wfExecutor.HasError()
+	}
```

Comment:
could this part be simplified?
all we need is just
```
	if wfExecutor.Template.Resource != nil {
		err = wfExecutor.ReportOutputsLogs(bgCtx, artifacts)
```
right?

Reply:
Yes, it can be simplified and kept consistent with the others.